cpp/dolfinx/fem/utils.cpp

```diff
@@ -445,12 +445,14 @@ fem::create_element_dof_layout(const ufc_dofmap& dofmap,
             num_entity_dofs.data());
 
   // Fill entity dof indices
+  int dof_count = 0;
```

Comment:
Where is this variable used?

Reply:
it must've been used in some code I've since deleted